docs/release_notes/6.0.9-7.md

```diff
@@ -0,0 +1,28 @@
+# Percona Server for MongoDB 6.0.9-7 (2023-09-13)
+
+[Installation](../install/index.md){.md-button}
+
+
+Percona Server for MongoDB 6.0.9-7 is an enhanced, source-available, and highly-scalable database that is a
+fully-compatible, drop-in replacement for MongoDB 6.0.9 Community Edition.
+
+It is based on [MongoDB 6.0.9 Community edition](https://www.mongodb.com/docs/v6.0/release-notes/6.0/#6.0.9---aug-14--2023) and supports the upstream protocols and drivers.
+
+
+## Release Highlights
+
+* The ability to [configure AWS STS endpoint](../aws-iam-setup.md#configure-aws-sts-endpoint) improves authentication and connectivity with AWS services.
```

Comment:
I thought this ability was added in previous release (with PSMDB-1291).

Reply:
Updated, thanks for the catch!